package.json

```diff
@@ -25,6 +25,7 @@
     "express": "^4.17.2",
     "moment": "^2.29.1",
     "react": "^17.0.2",
+    "react-bootstrap": "^2.0.4",
```

Comment:
We had a conversation about this dependency and my takeaway was that it's very limited compared to the vanilla bootstrap library that Sam imported

Reply:
got it. uninstalling now